yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher.java

```diff
@@ -326,16 +333,21 @@ public DockerComputerLauncher getPreparedLauncher(String cloudId, DockerSlaveTem
         cloneJNLPlauncher.setUser(getUser());
         cloneJNLPlauncher.setJvmOpts(getJvmOpts());
         cloneJNLPlauncher.setSlaveOpts(getSlaveOpts());
+        cloneJNLPlauncher.setTunnel(getTunnel());
         cloneJNLPlauncher.setJenkinsUrl(getJenkinsUrl());
         cloneJNLPlauncher.setNoCertificateCheck(isNoCertificateCheck());
         cloneJNLPlauncher.setNoReconnect(isNoReconnect());
 
         return cloneJNLPlauncher;
     }
 
+    public String getTunnel() {
+        return tunnel;
+    }
+
     @Override
     public ComputerLauncher getLauncher() {
-        return new JNLPLauncher();
+        return new JNLPLauncher(tunnel, jvmOpts);
```

Comment:
Change here is not needed, real launcher is returned in method before.

Reply:
so for rendering options on jnlp url they use delegated launcher instead of root launcher now?